src/mouse_events.cpp

```diff
@@ -349,6 +362,17 @@ unit_map::const_iterator mouse_handler::find_unit(const map_location& hex) const
 	return pc_.gamestate().board_.find_visible_unit(hex, viewing_team());
 }
 
+/**  Wraps 'const map_location game_display::hex_clicked_on(int, int)
+ * const'. Refer there for further documentation. */
+const map_location mouse_handler::hex_clicked_on(int x, int y) const {
+	return gui_->hex_clicked_on(x, y);
+}
+
+/**  Unit exists on the hex, no matter if friend or foe. */
+bool mouse_handler::hex_hosts_unit(const map_location& hex) const {
```

Comment:
Please put the leading function brace on its own line and remove excessive indent in comment. Since this is a short comment it's alright to have it all on one line.

Reply:
Will comply request.